Documentation/ApiOverview/Environment/ApplicationContext.rst

```diff
@@ -0,0 +1,383 @@
+:orphan:
+
+..  include:: /Includes.rst.txt
+..  index:: ! Application Context
+
+..  _application-context:
+
+===================
+Application Context
+===================
+
+In previous TYPO3 versions, error messages were displayed unfiltered in the
```

Comment:
Which "previous TYPO3 version"? AFAIK these are all out of LTS or even ELTS, so maybe "outdated TYPO3 version" or just name them "TYPO3 versions below 9" or something to avoid confusion.

Reply:
Thank you. Will be updated with next push